python/GangaDirac/Lib/Files/DiracFile.py

```diff
@@ -174,8 +174,7 @@ def _attribute_filter__set__(self, name, value):
         if value != "" and value is not None:
             #   Do some checking of the filenames in a subprocess
             if name == 'lfn':
-                if self.namePattern == '':
-                    self.namePattern = os.path.basename(value)
+                self.namePattern = os.path.basename(value)
```

Comment:
I'm probably forgetting how `namePattern` is used, but just overwriting it with LFN, won't this stomp over wildcards such as:
```
DiracFile.namePattern = '*.root'
DiracFile.lfn = 'hello.root'
```

I guess this may not not be a bad thing but won't it break copies of the `DiracFile` preserving the name pattern used for matching?